components/tag-list/tag-list-item-mixin.js

```diff
@@ -200,13 +200,14 @@ export const TagListItemMixin = superclass => class extends LocalizeCoreElement(
 		));
 	}
 
-	_renderTag(tagContent, hasTruncationTooltip) {
+	_renderTag(tagContent, hasTruncationTooltip = false, description = '') {
```

Comment:
Do you need the default parameters here?

Reply:
Nope 👍🏻